mycroft/dialog/__init__.py

```diff
@@ -47,18 +47,20 @@ def load_template_file(self, template_name, filename):
         """
         with open(filename, 'r') as f:
             for line in f:
-                template_text = line.strip()
-                if template_name not in self.templates:
-                    self.templates[template_name] = []
-
-                # convert to standard python format string syntax. From
-                # double (or more) '{' followed by any number of whitespace
-                # followed by actual key followed by any number of whitespace
-                # followed by double (or more) '}'
-                template_text = re.sub(r'\{\{+\s*(.*?)\s*\}\}+', r'{\1}',
-                                       template_text)
-
-                self.templates[template_name].append(template_text)
+                # Skip all lines starting with '#'
+                if not line.startswith('#'):
```

Comment:
I'd like to do the line.strip() before this check, there is no reason to be that whitespace sensitive.  So something like:
```python
    template_text = line.strip()
    if not template_text.startswith('#'):
        ...
```

Reply:
Good call. I'll do the change immediately